build/ci/windows/sign.bat

```diff
@@ -0,0 +1,66 @@
+@echo off
+ECHO "MuseScore sign"
+
+SET CERT_ENCRYPT_SECRET=""
+SET CERT_PASSWORD=""
+SET FILES_DIR=""
+SET FILE_PATH=""
+SET FILE_NAME=""
+
+SET SIGNTOOL="C:\Program Files (x86)\Windows Kits\10\bin\x64\signtool.exe"
+SET SIGNCERT="build\ci\windows\resources\musescore.p12"
+
+::SET TIMESERVER1="http://timestamp.globalsign.com/scripts/timstamp.dll"
+SET TIMESERVER1="http://timestamp.verisign.com/scripts/timstamp.dll"
+SET TIMESERVER2="http://timestamp.sectigo.com"
+SET TIMESERVER3="http://tsa.starfieldtech.com"
```

Comment:
This one does not provide an https page, do we really want to pass username/password across such a connection?

Reply:
I will test, I think it works without any username and password, because timestamp works on other port, not default 80 or 443

Reply:
I did not understand the question first. As far as I see, through this connection, nothing is transmitted, only get a timestamp

Reply:
Not if that port isn't getting explicity specified on the URL, like http://tsa.starfieldtech.com:13 (daytime) or http://tsa.starfieldtech.com:37 (time) or http://tsa.starfieldtech.com:123 (ntp). No need to try: neither of these work ;-)

Reply:
I think the signtool.exe utility can open the TCP connection to the right port on this url. ie the port determines the specification of obtaining a timestamp

Reply:
Explicit specify the port needs for the browser, because it does not know what we want from it :)

Reply:
The signtool probably needs just hostnames, not the protocol, so the entire "http://" stuff should not be needed

Reply:
Hmm, not sure, it claims that the `/t` option takes a URL as an additional parameter, not just a plain hostname?

Reply:
Yes, I agree. just so used to work and so publish the urls

Reply:
Ther example at https://docs.microsoft.com/de-de/windows/win32/seccrypto/adding-time-stamps-to-previously-signed-files says: `igntool timestamp -t https://timestamp.digicert.com MyControl.exe` so a) really a URL and b) using https ;-)

Reply:
In any case, I will check the work of each address :)